qiskit/quantum_info/analysis/distance.py

```diff
@@ -18,7 +18,7 @@ def hellinger_fidelity(dist_p, dist_q):
     """Computes the Hellinger fidelity between
     two counts distributions.
 
-    The fidelity is defined as 1-H where H is the
+    The fidelity is defined as (1-H**2)**2 where H is the
```

Comment:
Is there a reference for this we can include?

Reply:
I can like to a couple of wiki pages where the correspondence is clear.